lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -110,6 +111,22 @@ llvm::Error setTensorType(const caffe2::QTensorProto &in, Tensor *T) {
     RETURN_ERR("Only uint8 and int32 qtensors are supported");
   }
 }
+
+Storage *createPlaceholderFromWeightTensor(
+    PlaceholderBindings *bindings, Function &f, Tensor &weights,
+    const std::string &name, bool allowMove, bool isQuantized,
```

Comment:
How are you deciding when to use `allowMove`?  In what circumstances do we want to do that? It looks a bit arbitrary to me when you have decided to move vs. not, e.g. when loading a Conv you do it for bias but not filter.

Reply:
locally created tensors can be moved.